manifests/v2/base/webhook/manifests.yaml

```diff
@@ -0,0 +1,129 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook-configuration
```

Comment:
I am wondering for what are we planning to use Mutating Webhook ?

Reply:
IIUC, we need to default .spec.mlPolicy.numNodes to 1, right?

Reply:
Yeah, but can we set these values via CEL or we require Mutating Webhook in any case if we want to add default values?

Reply:
That makes sense.  In that case, let's decide if we should introduce the webhook mutator during https://github.com/kubeflow/training-operator/issues/2209.

So, we can remove the mutator for now.

Reply:
Done.